[2025-04-13T19:53:36.696+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T19:53:37.500+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.download_file scheduled__1997-01-01T00:00:00+00:00 [queued]>
[2025-04-13T19:53:37.733+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.download_file scheduled__1997-01-01T00:00:00+00:00 [queued]>
[2025-04-13T19:53:37.736+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-04-13T19:53:38.118+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_file> on 1997-01-01 00:00:00+00:00
[2025-04-13T19:53:39.218+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gcs_dlt_data_pipeline', 'download_file', 'scheduled__1997-01-01T00:00:00+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/first_stage/first_stage.py', '--cfg-path', '/tmp/tmpyxtn1d68']
[2025-04-13T19:53:39.347+0000] {standard_task_runner.py:105} INFO - Job 209: Subtask download_file
[2025-04-13T19:53:39.594+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=276) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-13T19:53:39.613+0000] {standard_task_runner.py:72} INFO - Started process 310 to run task
[2025-04-13T19:53:41.366+0000] {task_command.py:467} INFO - Running <TaskInstance: gcs_dlt_data_pipeline.download_file scheduled__1997-01-01T00:00:00+00:00 [running]> on host 0728476a73c7
[2025-04-13T19:53:45.269+0000] {job.py:229} INFO - Heartbeat recovered after 12.02 seconds
[2025-04-13T19:53:45.284+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_dlt_data_pipeline' AIRFLOW_CTX_TASK_ID='download_file' AIRFLOW_CTX_EXECUTION_DATE='1997-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1997-01-01T00:00:00+00:00'
[2025-04-13T19:53:45.315+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-13T19:53:45.803+0000] {logging_mixin.py:190} INFO - Source File Path: raw/1997.csv.bz2
[2025-04-13T19:53:45.804+0000] {logging_mixin.py:190} INFO - Destination File Path: /tmp/1997.csv.bz2
[2025-04-13T19:53:45.854+0000] {logging_mixin.py:190} INFO - Downloading raw/1997.csv.bz2 from bucket db_project_data_bucket to /tmp/1997.csv.bz2
[2025-04-13T19:54:54.704+0000] {logging_mixin.py:190} INFO - Downloaded db_project_data_bucket/raw/1997.csv.bz2 to /tmp/1997.csv.bz2
[2025-04-13T19:56:20.826+0000] {job.py:229} INFO - Heartbeat recovered after 25.21 seconds
[2025-04-13T19:56:28.242+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-04-13T19:56:29.443+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T19:56:30.086+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-13T19:56:30.407+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
